modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/VersionResolver.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.introspection;
+
+import static org.mule.config.i18n.MessageFactory.createStaticMessage;
+import org.mule.api.MuleRuntimeException;
+import org.mule.extension.annotations.Extension;
+
+/**
```

Comment:
This comment is wrong.. The VersionResolver should only be about resolving the version. Whether it checks a manifest or not is something specific to each implementation